eng/templates/stages/mirror.yml

```diff
@@ -36,8 +31,8 @@ stages:
       value: dn-bot@microsoft.com
     - name: destinationUrl
       value: https://dotnet-security-partners@dev.azure.com/dotnet-security-partners/dotnet/_git/dotnet
-    - name: sdkVersion
```

Comment:
If I understand things correctly, this is changing the tagging for 6.0/7.0 as well.  I think we should retain using the sdk version for the servicing branches.

Reply:
I changed it so that non-custom-tagged non-preview releases keep the SDK version. But I am not sure what behaviour should we have for 8.0 servicing branches. Should this only happen for 6.0/7.0 then?